packages/showcases/src/pages/e-commerce.tsx

```diff
@@ -256,52 +187,192 @@ export default function Home() {
+                    </p>
+                </div>
+            </div>
+        )
+    }
+
 
     return (
-        <>
+        <div className="px-6">
             <Head>
                 <title>Shop The Look</title>
                 <link rel="icon" href="/favicon.ico"/>
             </Head>
 
+            <div className="mt-6">
+                <p className="text-sm text-gray-500 mb-2">Search fashion products with an image+description</p>
+                <Dropzone onDrop={acceptedFiles => {
+                    setOriginalDocuments(acceptedFiles)
+                    search(addDesc, ...acceptedFiles)
+                }}>
+                    {({getRootProps, getInputProps}) => (
+                        <div {...getRootProps()}
+                             className="border-b-0 cursor-pointer border border-primary-500 rounded-t flex flex-col items-center p-8 ">
+                            <div className="h-8 w-8 mb-3">
+                                <Image src={SearchIcon}/>
+                            </div>
+                            <input {...getInputProps()} />
+                            <p className="font-bold w-72 text-center">Drag and drop your image here or <span
+                                className="text-primary-500">Browse</span> to select a file
+                            </p>
+                            <div className="flex items-center">
+                                <div className="w-6 h-6">
+                                    <Image src={Picture}/>
+                                </div>
+                                <p className="text-gray-500 text-sm">Limit 200 MB per file</p>
+                            </div>
+                        </div>
+                    )}
+                </Dropzone>
+                <input
+                    onChange={(event) => setAddDesc(event.target.value)}
+                    className="textInput appearance-none block w-full text-gray-700 border border-primary-500 rounded-b py-3 px-4 mb-3 leading-tight focus:outline-none focus:bg-white"
+                    id="grid-first-name" type="text" placeholder="Add additional description"/>
+                <style jsx>
+                    {`
+                      .textInput {
+                        border-top-color: #E5E5E5;
+                      }
+                    `}
+                </style>
+            </div>
+
+            <Filters onFilter={onFilter}/>
 
-            <SearchBar search={handleSearch} searching={searching}/>
-            <Filters onFilter={filter}/>
-            <div className="h-6 text-red-500">
-                {error}
+            <h2 className="mt-12 font-bold mb-6">Click on example queries:</h2>
+            <div className="flex justify-between px-12">
+                <ExampleQuery/>
+                <ExampleQuery color="black"/>
+                <ExampleQuery color="white"/>
+                <ExampleQuery color="yellow"/>
```

Comment:
For image + yellow and image + green, I'm not getting any results. Could you put example queries that gives results?